packages/base/src/formbuilder/objectform/dissolveProcessForm.tsx

```diff
@@ -0,0 +1,67 @@
+import { BaseForm, IBaseFormProps } from './baseform'; // Import BaseForm
+import { IDict, IJupyterGISModel, IGeoJSONSource } from '@jupytergis/schema';
+import { IChangeEvent } from '@rjsf/core';
+import { loadFile } from '../../tools';
+
+interface IDissolveFormOptions extends IBaseFormProps {
+  schema: IDict;
+  sourceData: IDict;
+  title: string;
+  cancelButton: (() => void) | boolean;
+  syncData: (props: IDict) => void;
+  model: IJupyterGISModel;
+}
+
+export class DissolveFormDialog extends BaseForm {
```

Comment:
Since this is not a dialog, I suggest you rename it DissolveForm :)

```suggestion
export class DissolveForm extends BaseForm {
```